### ISSUE TITLE
AFTER BP UPDATE AUTOMATIC DISCONNECTION FROM THE BEGINNING OF A MATCH

### ISSUE THREAD

**Issue by Fix-it3** [ValveSoftware/Dota2-Gameplay#5772]
### Description

ATFER THE 2022 BP UPDATE,MANY PLAYERS ARE HAVING AUTOMATIC DISCONNECTION ISSUE AT THE BEGINNING OF THE MATCH.THIS PROBLEM IS NOT BECAUSE OF NETWORK PROBLEM BECAUSE WE TRIED OTHER WAYS TO TEST NETWORK PROBLEM BUT THE PROBLEM STILL REMAINS,ALSO NO FPS ISSUE.IF YOU WANNA SEE PROOF THEN CHECK HOW MANY PLAYERS ARE GETTING DISCONNECTED FROM MATCHES.!!!SPECIALLY!!! SINGLE DRAFT.PLEASE FOX THIS ASAP BECAUSE THOSE WHO ARE FACING THIS PROBLEM CAN'T GET OUT OF LP and PLEASE """"REMOVE THE BUG"""".THANK YOU

**Comment by Purumal** [comment]
I have noticed this a lot in AD recently. But do you have some kind of evidence that it is due to the BP?

**Comment by Fix-it3** [comment]
> I have noticed this a lot in AD recently. But do you have some kind of evidence that it is due to the BP?



> I have noticed this a lot in AD recently. But do you have some kind of evidence that it is due to the BP?

**Comment by Fix-it3** [comment]
> I have noticed this a lot in AD recently. But do you have some kind of evidence that it is due to the BP?

yes,I have been playing Dota2 for 6 years.and I was a active dota player before BP update.But just after the BP I started facing this issue.
